docs/website/docs/plus/features/projects.md

```diff
@@ -190,6 +189,89 @@ As you may guess from the example above, you can use Python-style formatters to
 * You can reference environment variables using the `{env.ENV_VARIABLE_NAME}` syntax.
 * Any of the project settings can be substituted as well.
 
+### implicit entities
+By default, dlt+ will automatically create entities such as datasets or destinations when they are requested by the user or the executed code.
+For example, a minimal `dlt.yml` configuration might look like this:
+```yaml
+sources:
+  arrow:
+    type: sources.arrow.source
+
+destinations:
+  duckdb:
+    type: duckdb
+
+pipelines:
+  my_pipeline:
+    source: arrow
+    destination: duckdb
+    dataset_name: my_pipeline_dataset
+```
+Running the following command executes the pipeline:
+```sh
+dlt pipeline my_pipeline run
+```
+In this case, the `my_pipeline_dataset` dataset is not declared explicitly, so dlt+ creates it automatically. The same applies to the `duckdb` destination if it is not defined outside of the pipeline configuration.
+
+Implicit creation of entities can be controlled using the `allow_undefined_entities` setting in the project configuration:
+
+```yaml
+project:
+  allow_undefined_entities: false
+```
+If `allow_undefined_entities` is set to `false`, dlt+ will no longer create missing entities automatically. 
+Datasets and destinations must be declared explicitly in the `dlt.yml` file:
+```yaml
+datasets:
+  my_pipeline_dataset:
+    destination:
+        - duckdb
+```
+:::Note
+The destination field is an array, allowing you to specify one or more destinations where the dataset can be materialized. 
+:::
+
+### Managing Datasets and Destinations
+
+When datasets are explicitly declared in the `dlt.yml` file, the `destination` field must list all destinations where the dataset is allowed to be materialized. This applies even if `allow_undefined_entities` is set to `true`.
+Example:
+```yaml
+sources:
+  arrow:
+    type: sources.arrow.source
+
+destinations:
+  duckdb:
+    type: duckdb
+
+
+datasets:
+  my_pipeline_dataset:
+    destination:
+        - bigquery
+
+pipelines:
+  my_pipeline:
+    source: arrow
+    destination: duckdb
+    dataset_name: my_pipeline_dataset
+```
+In the above configuration, the dataset `my_pipeline_dataset` includes only `bigquery` as an allowed destination. If the pipeline specifies `duckdb` as the destination, dlt+ will return an error because `duckdb` is not listed for that dataset.
+All intended destinations must be listed explicitly in the `destination` field to avoid configuration errors.
+
+```yaml
+datasets:
+  my_pipeline_dataset:
+    destination:
+      - duckdb
+      - bigquery
+```
+:::tip
+If no destination is specified in a CLI command, dlt+ will use the first listed destination for that dataset by default.
+If implicit entities are allowed, dlt+ can also discover datasets defined only within pipelines and will use the pipeline's destination when creating the dataset.
```

Comment:
honestly i'm not sure this tip should be here at all, i don't get how it's relevant 🤔